.travis.yml

```
@@ -5,7 +5,7 @@ node_js:
   - 'lts/*'
   - '10'
 cache: npm
-script: npm test
+script: npm run lint && npm test
```

This while give slightly cleaner CI output:

```suggestion
script:
  - npm run lint
  - npm test
```